model_compression_toolkit/target_platform_capabilities/targetplatform2framework/attach2keras.py

```diff
@@ -104,13 +103,7 @@ def __init__(self):
             OperatorSetNames.L2NORM: [tf.math.l2_normalize],
         }
 
-        if FOUND_SONY_CUSTOM_LAYERS:
-            self._opset2layer[OperatorSetNames.SSD_POST_PROCESS] = [SSDPostProcess]
-        else:
-            # If Custom layers is not installed then we don't want the user to fail, but just ignore custom layers
-            # in the initialized framework TPC
-            self._opset2layer[OperatorSetNames.SSD_POST_PROCESS] = []
-
+        self._opset2layer[OperatorSetNames.SSD_POST_PROCESS] = [SSDPostProcess]
```

Comment:
you can add this to the dict initialization like all other operators, now that the `FOUND_SONY_CUSTOM_LAYERS` is removed